scripts/setup-k8s-master.sh.in

```diff
@@ -120,13 +140,10 @@ kubectl apply -f {{NetworkPolicyUrl}}
 
 # Use kubeadm's kubeconfig command to grab a client-cert-authenticated
 # kubeconfig file for administrative access to the cluster.
-# Because kubeadm doesn't set controlPlaneEndpoint as master, the
-# --apiserver-advertise-address is set to the load balancer external ipv4 address.
 KUBECONFIG_OUTPUT=/home/ubuntu/kubeconfig
 
-kubeadm alpha phase kubeconfig user \
+kubeadm alpha kubeconfig user \
```

Comment:
This command had to be updated as it would not run with the updated `kubeadm`

Reply:
Updated the wording from the comment I removed (I do think now that we need this to properly form the config for use outside the cluster).

As written above, the apostrophe is actually `\u2019` which is a right, single quote mark and causes the template to explode.